packages/terra-profile-image/src/ProfileImage.jsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import 'terra-base/lib/baseStyles';
+import TerraImage from 'terra-image';
+import avatarImage from './avatar.svg';
+import './ProfileImage.scss';
+
+const propTypes = {
+  /**
+   * The profile image to load.
+   */
+  image: PropTypes.element.isRequired,
+};
+
+const avatarOverrideProps = { src: avatarImage, onLoad: undefined, onError: undefined };
+
+class ProfileImage extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = { isLoading: true, isError: false };
+
+    this.updateImage(props.image.props);
+  }
+
+  componentWillReceiveProps(newProps) {
+    if (newProps === this.props) {
```

Comment:
I don't think this will work. newProps and this.props are different objects I believe, meaning this will always return false.